build-docs.sh

```
@@ -0,0 +1,28 @@
+
+GIT=$(git rev-parse HEAD)
+rm -rf ./dist/fireproof-docs
+git clone $(pwd)/.git -b docs ./dist/fireproof-docs
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)